inji-web/src/pages/IssuerPage/CertificateList.js

```diff
@@ -37,7 +37,7 @@ const getCardsData = (issuerId, issuerDisplayName, authEndpoint, credentialList,
                         issuerDisplayName,
                         certificateId: cred.id,
                         codeVerifier: codeVerifier,
-                        state: state,
+                        state: constructNewState(state),
```

Comment:
@vijay151096 We should only pass state here instead of the array. And we can update the existing state in local storage with the newly constructed state array in local storage.

Reply:
@shiva-beehyv , we are passing only that state to esignet not the entire array